spoon/template/compiler.php

```diff
@@ -775,28 +775,28 @@ protected function parseOptions($content)
 						$replace[] = '<?php
 						if(
 							(is_object(' . $baseVariable . ') && ' . $objectVariable . ' && ' . $objectVariable . ' != \'\' && ' . $objectVariable . ' !== false)
-							|| (is_array(' . $baseVariable . ') && isset(' . $variable . ') && count(' . $variable . ') != 0 && ' . $variable . ' != \'\' && ' . $variable . ' !== false))
```

Comment:
did this one actually give errors? cause the first one is an is_array so non arrays shouldn't get to the count part

Reply:
This gives an error. Just checked by replacing it with the original and I get errors on de $headers variable with the datagrid.